proposals/distribution.md

```diff
@@ -107,16 +108,10 @@ The following entries should be added to the [scope table][scope]:
     Retrieving images covers the current “tag listing” (e.g. “what named manifests are in `library/busybox`?”), because tags are entries in the image format's [`manifests` array][manifests].
     Other tag-listing endpoints needed for backwards-compatibility are therefore in scope as well.
 
-    Repository actions and listing images within a repository are considered part of distribution policy or content management and are out of scope for this entry's per-image action.
-    For example, “what images are under `library/`?” is out of scope for this project.
-
     * What: Specifying a distribution method
     * In/Out/Future: In scope
     * Status: In progress (see opencontainers/distribution-spec)
     * Description: Define a protocol for image, manifest, config, and blob creation, retrieval, and deletion.
-        Listing repositories is a multi-repository action, which is out of scope for this entry.
-        Creating and deleting repositories are per-repository actions, which are out of scope for this entry.
-        Listing images within repositories is a per-repository action, which is out of scope for this entry.
```

Comment:
I'd rather keep these limits to ensure we stay focused on push/pull for images and more granular objects.   If the OCI wants to cover content-management at the repository level, that's fine with me, but I'd like it to be clearly separated from distribution so implementations could be compliant distribution engines without needing to implement repository-level operations. However, I'm happy to take that distinction up with the coming distribution maintainers if the TOB wants to punt on it by removing these lines here.  More discussion in this vein in #37 and #44.

Reply:
Just as enabling content-management is not the same as providing content-management, I don't think providing a list of contents to enable certain push and pull workflows is the same as "covering content-management." 

Per the discussions over on 44.. If there's a phrasing you want to add to ensure the resulting spec delineates MUST items necessary to certain primary workflows and further delineates as OPTIONAL the portions of the spec not necessary for the primary workflows, that works for me.

Reply:
> Per the discussions over on #44.

I'll keep my pushback over on #44 so other folks can weigh in here with a lower comment-count ;).